### ISSUE TITLE
transformClassesAndResourcesWithSyncLibJarsForDebug task fail with ktlint-gradle plugin

### ISSUE THREAD

**Issue by illuzor** [JLLeitschuh/ktlint-gradle#138]
I am trying to run _gradlew connectedCheck_ task, but it fails. After removing  ktlint-gradle it finishes successful.

Screenshot 1 - [fail](https://images.illuzor.com/uploads/ktlint-plugin-error.png)
Screenshot 2 - [disabled plugin and success](https://images.illuzor.com/uploads/ktlint-plugin-off-no-error.png)

Plugin version - 6.0.0

Log text:

```
Task :klib:transformClassesAndResourcesWithSyncLibJarsForDebug FAILED
file or directory 'D:\_sources\AndroidStudio\my_projects\Klib\klib\libs', not found
file or directory 'D:\_sources\AndroidStudio\my_projects\Klib\klib\libs', not found
Task ':klib:transformClassesAndResourcesWithSyncLibJarsForDebug' is not up-to-date because:
  Input property '$3' file D:\_sources\AndroidStudio\my_projects\Klib\klib\build\intermediates\javac\debug\compileDebugJavaWithJavac\classes has been removed.
  Input property '$3' file D:\_sources\AndroidStudio\my_projects\Klib\klib\build\intermediates\javac\debug\compileDebugJavaWithJavac\classes\androidx has been removed.
  Input property '$3' file D:\_sources\AndroidStudio\my_projects\Klib\klib\build\intermediates\javac\debug\compileDebugJavaWithJavac\classes\androidx\appcompat has been removed.
file or directory 'D:\_sources\AndroidStudio\my_projects\Klib\klib\libs', not found
file or directory 'D:\_sources\AndroidStudio\my_projects\Klib\klib\libs', not found
:klib:transformClassesAndResourcesWithSyncLibJarsForDebug (Thread[Task worker for ':' Thread 4,5,main]) completed. Took 0.004 secs.

FAILURE: Build failed with an exception.

 What went wrong:
Execution failed for task ':klib:transformClassesAndResourcesWithSyncLibJarsForDebug'.
```

**Comment by JLLeitschuh** [comment]
Can you run the task with `--stacktrace` flag and paste the stack trace you get?

**Comment by illuzor** [comment]
https://gist.github.com/illuzor/34ba2717ff17f6d9c0428cff27a41f78

**Comment by Tapchicoma** [comment]
@illuzor I suppose ktlint plugin version is `6.0.0`?

If yes - can you try `5.1.0` version of the plugin?

**Comment by Tapchicoma** [comment]
Most probably related to #135

**Comment by Tapchicoma** [comment]
Please try released version `6.1.0` and feel free to reopen this issue.

**Comment by illuzor** [comment]
@Tapchicoma thank you for quick fix. No more problem
